packages/core/src/browser/context-menu-renderer.ts

```diff
@@ -24,5 +26,34 @@ export function toAnchor(anchor: HTMLElement | { x: number, y: number }): Anchor
 
 export const ContextMenuRenderer = Symbol('ContextMenuRenderer');
 export interface ContextMenuRenderer {
+    render(options: RenderContextMenuOptions): void;
```

Comment:
I was expecting exactly such an API a few weeks ago, but registering the commands and the menu items also did the trick.